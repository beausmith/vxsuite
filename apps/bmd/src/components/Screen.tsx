import styled from 'styled-components'

const Screen = styled.div`
  height: 100%;
  display: flex;
  flex-direction: column;
`
export default Screen
