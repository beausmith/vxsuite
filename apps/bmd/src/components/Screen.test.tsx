import { axe } from 'jest-axe'
import React from 'react'
import { render } from '../../test/testUtils'

import Screen from './Screen'

it(`renders Screen`, async () => {
  const { container } = render(<Screen>foo</Screen>)
  expect(await axe(container.innerHTML)).toHaveNoViolations()
  expect(container.firstChild).toMatchSnapshot()
})
