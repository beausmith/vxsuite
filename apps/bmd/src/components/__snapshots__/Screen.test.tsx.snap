// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Screen 1`] = `
.c0 {
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  class="c0"
>
  foo
</div>
`;
